vector/src/main/java/im/vector/app/core/ui/views/TypingMessageAvatar.kt

```diff
@@ -34,19 +34,22 @@ class TypingMessageAvatar @JvmOverloads constructor(
 ) : LinearLayout(context, attrs, defStyleAttr) {
 
     companion object {
-        const val AVATAR_SIZE_DP = 20
         const val OVERLAP_FACT0R = -3 // =~ 30% to left
     }
 
+    private val typingAvatarSize by lazy {
```

Comment:
if render is always called from the main thread we could use the unsafe lazy initializer to avoid a synchronised call